ci_build_images/msan.Dockerfile

```diff
@@ -0,0 +1,29 @@
+# this is to create images with MSAN for BB workers
+WORKDIR /tmp/msan
+
+COPY build-msan2.sh .
+
+RUN curl -sL https://apt.llvm.org/llvm-snapshot.gpg.key | gpg --dearmor -o /usr/share/keyrings/llvm-snapshot.gpg \
+    && echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/llvm-snapshot.gpg] \
+    http://apt.llvm.org/jammy/ llvm-toolchain-jammy-15 main" > /etc/apt/sources.list.d/llvm-toolchain.list \
+    && echo "deb-src [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/llvm-snapshot.gpg] \
+    http://apt.llvm.org/jammy/ llvm-toolchain-jammy-15 main" >> /etc/apt/sources.list.d/llvm-toolchain.list \
+    && apt-get update \
+    && echo 'APT::Get::Assume-Yes "true";' > /etc/apt/apt.conf.d/90forceyes \
+    && apt-get -y install --no-install-recommends dpkg-dev devscripts clang-15 \
```

Comment:
`dpkg-dev` and `devscripts` are already in the image.

Reply:
Deleted.